commands/music/np.js

```diff
@@ -0,0 +1,39 @@
+/* eslint linebreak-style: 0 */
+const Command = require(`${process.cwd()}/base/Command.js`);
+const { MessageEmbed } = require("discord.js");
+
+class NP extends Command {
+  constructor(client) {
+    super(client, {
+      name: "np",
+      description: "This command will display the current playing song.",
+      usage: "np",
+      category: "Music",
+    });
+  }
+
+  async run(message, args, level) { // eslint-disable-line no-unused-vars
+    const voiceChannel = message.member.voiceChannel;
+
+    const noVoiceChannelEmbed = new MessageEmbed()

```

Comment:
Lines 23 and 29.  You really need to remove duplication of variables and unnecessary embeds.